### DISCUSSION TITLE
It's hard to contact users in a young enterprise

### DISCUSSION THREAD

**Discussion by jessehouwing** [community/community#106326] [Enterprise]
A number of our organizations have recently been upgraded from a legacy license to Enterprise and we're rolling out new features that have become available.

It's been a really big pain to reach out to users who might be impacted by the changes we're rolling out.

I'd love a number of features in this area:

 * `Contact member(s)` button at the Organization/Members section. After selecting everyone without 2FA, allow me to contact them. Same for SSO, same for outside collaborators etc. Or to create an issue for them or to create a new discussion that tags these users.
 * Similarly, from the "notifications email" domain enforcement pop-up I can copy/paste the list of users, but there is no easy way to contact these users easily. I can copy/paste from the screen, that gives me their github handles, but no easy way to reach out to them.
 * The CSV and JSON downloads from the members section contains an SSO email, but not their enterprise notification email.
 * There is no easy way to target all repo admins that use a certain feature. Requires all kinds of scripting.
 * There is no way to invite people to associate their SSO, except to throw them out of the org and invite them back in.
 * None of the notification emails for invitations, removals, SSO enforcement, 2FA enforcement contain options to customize the notification. No way to link to documentation, not way to provide a point of contact.
 * The Team Discussions feature that was removed can't fully be replaced by discussions, discussions are way less private than reaching out to a secret team.
 * Depending on where you copy t a list of handles from they have an `@` or not. Copy them from the UI in the notifications domain pop-up and you get an `@`, copy them from the members CSV export and there isn't any. This makes pasting a list of users into an issue not a viable solution.
 * There is no `gh-cli` feature to do any of this either.
 * Ideally there would be an enforcement option which requires users to add their notification email and custom route next time they visit the org.
 * When enabling SSO, you can't reach out to the impacted users.
 * Especially contacting users that do NOT have an enterprise-owned domain can be troublesome, I get that, but it's not hard to grab their public email from the API, so it's not really a blocker for email harvesting or spam prevention as far as I can tell.
 * Contacting `@everyone` or `@org/repo` isn't possible. Nor can I easily tag these in a discussion. 
 * I can't find a way to query the preferred contact email for a user in my org. I can query their public email, their enterprise-owned domain emails and their SSO email. But can't query the custom route email for my org.
 * There is no clear way for a user to see who is the admin of a repo. You can see the contributors, but it would be super useful if organization members could see the admins of a repo so they know who to reach out to.

Ideally it would be A LOT easier for an enterprise/org admin to communicate to their org members.

Once users add their verified organization/enterprise email, a lot of these features should be available with the click of a button. I don't mind popping up outlook when I click such a link, but ideally github would send the mail in my behalf.

**Comment by ossanna16** [comment]
Thank you for the valuable feedback @jessehouwing!
